<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>609</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MPSoC Simulation Environment</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mpsoc.ico</normaloff>mpsoc.ico</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="leftPanel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="simulationPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="text">
          <string>Simulation</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/images/play-button.png</normaloff>:/icons/images/play-button.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applicationsPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="text">
          <string>Applications</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/images/network.png</normaloff>:/icons/images/network.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="heuristicsPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="text">
          <string>Heuristics</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/images/route.png</normaloff>:/icons/images/route.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="widgetStack">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionShow_Help"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>&amp;Simulation</string>
    </property>
    <addaction name="actionAdd_MPSoC"/>
    <addaction name="separator"/>
    <addaction name="action_Reset_Simulation"/>
    <addaction name="actionStep_Back"/>
    <addaction name="actionStep_Foward"/>
    <addaction name="actionAuto_Step"/>
   </widget>
   <widget class="QMenu" name="menuApplications">
    <property name="title">
     <string>&amp;Applications</string>
    </property>
    <addaction name="action_Open_Applications_Directory"/>
   </widget>
   <widget class="QMenu" name="menuHeuristics">
    <property name="title">
     <string>H&amp;euristics</string>
    </property>
    <addaction name="action_Open_Heuristics_Directory"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuApplications"/>
   <addaction name="menuHeuristics"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/exit-2.png</normaloff>:/icons/images/exit-2.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShow_Help">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/idea.png</normaloff>:/icons/images/idea.png</iconset>
   </property>
   <property name="text">
    <string>Show &amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/info.png</normaloff>:/icons/images/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionAdd_MPSoC">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/add-1.png</normaloff>:/icons/images/add-1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add MPSoC</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_Reset_Simulation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/repeat-1.png</normaloff>:/icons/images/repeat-1.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset Simulation</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStep_Back">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/previous.png</normaloff>:/icons/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>Step &amp;Back</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionStep_Foward">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/skip.png</normaloff>:/icons/images/skip.png</iconset>
   </property>
   <property name="text">
    <string>Step &amp;Foward</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionAuto_Step">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/play-button-1.png</normaloff>:/icons/images/play-button-1.png</iconset>
   </property>
   <property name="text">
    <string>Au&amp;to Step</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_Open_Applications_Directory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/folder-12.png</normaloff>:/icons/images/folder-12.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Directory</string>
   </property>
  </action>
  <action name="action_Open_Heuristics_Directory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/images/folder-12.png</normaloff>:/icons/images/folder-12.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>simulationPushButton</tabstop>
  <tabstop>applicationsPushButton</tabstop>
  <tabstop>heuristicsPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
